en/docs/assets/attachments/sample-sso-redirect-page-template.html

```diff
@@ -0,0 +1,41 @@
+<!-- 
+Variables $acUrl, $response, $relayState and $additionalParams will be replaced by the corrosponding values  
+-->
+<html>
+	<body>
+        	<div id="content1" style="visibility: hidden;">
+			
+			<p>Content Area for Service Provider One</p>
+
+		</div>
+                <div id="content2"  style="visibility: hidden;">
+
+			<p>Content Area for Service Provider Two</p>
+		</div>
+
+
+        	<form method='post' action='$acUrl'>
+       			<p>
+        			<input type='hidden' name='SAMLResponse' value='$response'>
+        			<input type='hidden' name='RelayState' value='$relayState'>
+                    <!--$additionalParams-->
+        			<button type='submit'>POST</button>
+       			</p>
+       		</form>
+
+                 <script type='text/javascript'>
+        		var url = '$acUrl';
+			
+			if(url === "https://localhost:9453/acs")
```

Comment:
Don't use hard coded urls. Instead use a placeholder and guide users on how to replace it with their value

Reply:
Will fix in the samples revisit iteration